.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: false
+sudo: required
```

Comment:
> The CNAME record solution let you deal with collisions. You'll get that from the AvahiEntryGroupState that is passed back to callbacks like http://avahi.org/download/doxygen/publish_8h.html#afbd8ce431f0416537fc7b6101e0f9a1c

Yeah I know, it just behaves weirdly for me - never complains about collision - will figure that out